ci-operator/step-registry/gcp/provision/c3-metal/network/gcp-provision-c3-metal-network-ref.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "gcp/provision/c3-metal/network/gcp-provision-c3-metal-network-ref.yaml",
```

Comment:
Okay, thanks for the confirm on the https://github.com/openshift/release/pull/58966#discussion_r1857693284, my above suggestion is still valid, so that we make the step to be more common to be re-used in more cases

Reply:
I prefer not using the keyword "shared-vpc", because it refers to specific concepts in GCP, see [Shared VPC](https://cloud.google.com/vpc/docs/shared-vpc) in GCP.

Reply:
Make sense, then how about `existing-vpc`

Reply:
"gcp/provision/c3-metal/existing-vpc"?

Reply:
"gcp/existing-vpc"

Reply:
Even those network/subnets are in the GCP C3 bare metal testing project?

Reply:
Even not under "gcp/provision"?

Reply:
> Even those network/subnets are in the GCP C3 bare metal testing project

I do not think the step usage is dedicated for c3 baremetal testing, just like the above comments you explained, we even can dynamically provision vpc using gcloud commands lines (if we want to develop it in gcp-provision-vpc step). Now sounds like you prefer to use the pre-existing vpc in the testing. The step also can be re-used in other similar install process. That is why I do not include `c3-metal` keyword.

> Even not under "gcp/provision"?

`provision` somehow means create some resources on the demand, but actually in this case, we just used an existing vpc, so I did not recommend to put them in `provision` folder.

Reply:
Got it, let me update. Thanks!

Reply:
@jianlinliu I happened noticing `ci-operator/step-registry/ipi/conf/gcp/byo-vpc/`, do you think we'd better to use the name `byo-vpc`  instead? Thanks!

Reply:
Sure, no problem.